tests/service.netcore/service.fsproj

```diff
@@ -0,0 +1,65 @@
+    <NoWarn>$(NoWarn);44</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DelaySign>true</DelaySign>
+    <DebugType>portable</DebugType>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>service</AssemblyName>
+    <PackageId>service</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
+    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="../service/ReshapedReflection.fs">
+      <Link>ReshapedReflection.fs</Link>
+    </Compile>
+    <Compile Include="../service/FsUnit.fs">
+      <Link>FsUnit.fs</Link>
+    </Compile>
+    <Compile Include="../service/Common.fs">
+      <Link>Common.fs</Link>
+    </Compile>
+    <Compile Include="../service/EditorTests.fs">
+      <Link>EditorTests.fs</Link>
+    </Compile>
+    <Compile Include="../service/ExprTests.fs">
+      <Link>ExprTests.fs</Link>
+    </Compile>
+    <Compile Include="../service/TokenizerTests.fs">
+      <Link>TokenizerTests.fs</Link>
+    </Compile>
+    <Compile Include="../service/PerfTests.fs">
+      <Link>PerfTests.fs</Link>
+    </Compile>
+    <Compile Include="../service/InteractiveCheckerTests.fs">
+      <Link>InteractiveCheckerTests.fs</Link>
+    </Compile>
+    <Compile Include="../service/ProjectAnalysisTests.fs">
+      <Link>ProjectAnalysisTests.fs</Link>
+    </Compile>
+    <None Include="App.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../src/fsharp/FSharp.Compiler.Service.netcore/FSharp.Compiler.Service.netcore.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170106-08" />
+    <PackageReference Include="NUnitLite" Version="3.6.1" />
+    <PackageReference Include="NUnit" Version="3.6.1" />
+    <PackageReference Include="FSharp.Core" Version="4.1.*" />
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
+    <PackageReference Include="System.IO" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Emit" Version="4.0.1" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.4.1" />
+    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.1.0" />
+    <PackageReference Include="System.Runtime.Loader" Version="4.0.0" />
+  </ItemGroup>
+  <Target Name="PrecompileScript" BeforeTargets="BeforeBuild">
+    <Exec Command="dotnet restore &quot;$(MSBuildProjectDirectory)/../projects/Sample_NETCoreSDK_FSharp_Library_netstandard1.6/Sample_NETCoreSDK_FSharp_Library_netstandard1.6.fsproj&quot;" />
+    <Exec Command="dotnet build &quot;$(MSBuildProjectDirectory)/../projects/Sample_NETCoreSDK_FSharp_Library_netstandard1.6/Sample_NETCoreSDK_FSharp_Library_netstandard1.6.fsproj&quot;" />
```

Comment:
using `'` will do better because you can use `"`

`<Exec Command="dotnet build "$(MSBuildProjectDirectory)/../projects/Sample_NETCoreSDK_FSharp_Library_netstandard1.6/Sample_NETCoreSDK_FSharp_Library_netstandard1.6.fsproj" ' />`